.travis.yml

```diff
@@ -36,7 +36,8 @@ env:
   - REQUIREMENTS=devel REXTRAS=docs
 
 python:
-  - "2.7"
+  - "3.3"
```

Comment:
Is there any value in testing older versions of Python? I'd like to have a faster travis response on pull requests.